libraries/service/src/main/java/org/holochain/androidserviceruntime/service/BootCompletedReceiver.kt

```diff
@@ -0,0 +1,30 @@
+package org.holochain.androidserviceruntime.service
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+
+/**
+ * BroadcastReceiver that starts the HolochainService when the device has finished booting.
+ */
+class BootCompletedReceiver : BroadcastReceiver() {
```

Comment:
Receives the `ACTION_BOOT_COMPLETED` intent from the system, then sends another Intent to the HolochainService.